packages/integration-tests/src/test-project-setup/cdk/test_cdk_project_creator.ts

```diff
@@ -0,0 +1,34 @@
+import { CloudFormationClient } from '@aws-sdk/client-cloudformation';
+import { e2eToolingClientConfig } from '../../e2e_tooling_client_config.js';
+import { TestCdkProjectBase } from './test_cdk_project_base.js';
+import { AuthTestCdkProjectCreator } from './auth_cdk_project.js';
+import { fileURLToPath } from 'url';
+import path from 'path';
+import { DeployedResourcesFinder } from '../../find_deployed_resource.js';
+
+const dirname = path.dirname(fileURLToPath(import.meta.url));
+export const testCdkProjectsSourceRoot = path.resolve(
+  dirname,
+  '..',
+  '..',
+  '..',
+  'src',
+  'test-cdk-projects'
```

Comment:
Was that on purpose? Why not...?

```suggestion
  '../../../src/test-cdk-projects'
```

Reply:
Yes, we run these tests on Windows too.
The recommended practice is to use `path`'s methods to join path segments.